indico_jacow/client/MailingList.jsx

```diff
@@ -0,0 +1,168 @@
+      if (response.status === 200) {
+        setNotSubMailingLists(prevLists =>
+          prevLists.map(list => (list.id === listsIds ? {...list, subscribed: false} : list))
+        );
+      }
+    } catch (e) {
+      console.error(e);
+    }
+  };
+
+  const handleSelectSubscribed = (e, {value}) => {
+    setSelectedSubscribed(prev =>
+      prev.includes(value) ? prev.filter(id => id !== value) : [...prev, value]
+    );
+  };
+
+  const handleSelectNotSubscribed = (e, {value}) => {
+    setSelectedNotSubscribed(prev =>
+      prev.includes(value) ? prev.filter(id => id !== value) : [...prev, value]
+    );
+  };
+
+  return (
+    <div style={{marginTop: '15px'}}>
+      {/* Subscribed Lists Section */}
+      <div className="i-box">
+        <div className="i-box-header">
+          <div className="i-box-title">
+            <Translate>Subscribed Lists</Translate>
+          </div>
+        </div>
+        <div className="i-box-content">
+          <List divided size="big">
+            {subMailingLists.map(list => (
+              <ListItem className="mailing" key={list.id}>
+                <ListContent>
+                  <Checkbox
+                    label={list.name}
+                    value={list.id}
+                    onChange={handleSelectSubscribed}
+                    checked={selectedSubscribed.includes(list.id)}
+                  />
+                </ListContent>
+              </ListItem>
+            ))}
+            <ListItem>
+              <Button
+                icon
+                labelPosition="left"
+                className="unsubscribe-btn"
+                onClick={() => unsubscribeList(selectedSubscribed)}
+                disabled={selectedSubscribed.length === 0}
+              >
+                <Icon name="minus" />
+                <Translate>Unsubscribe</Translate>
+              </Button>
+            </ListItem>
+          </List>
+        </div>
+      </div>
+
+      {/* Not Subscribed Lists Section */}
+      <div className="i-box" style={{marginTop: '15px'}}>
+        <div className="i-box-header">
+          <div className="i-box-title">
+            <Translate>Not Subscribed Lists</Translate>
+          </div>
+        </div>
+        <div className="i-box-content">
+          <List divided size="big">
+            {notSubMailingLists.map(list => (
+              <ListItem className="mailing" key={list.id}>
+                <ListContent>
+                  <Checkbox
+                    label={list.name}
+                    value={list.id}
+                    onChange={handleSelectNotSubscribed}
+                    checked={selectedNotSubscribed.includes(list.id)}
+                  />
+                </ListContent>
+              </ListItem>
+            ))}
+            <ListItem>
+              <Button
+                icon
+                labelPosition="left"
+                primary
+                className="subscribe-btn"
```

Comment:
For classes only defined in this file, can we use `styleName` instead of `className`? This lets you simplify the CSS selectors and avoid `:global()` ;)